site/docs/.vitepress/configs/locales/uk.ts

```diff
@@ -513,7 +513,7 @@ export const notFoundUk: Record<string, NotFound> = {
   uk: {
     title: "PAGE NOT FOUND",
```

Comment:
This too?

Reply:
I think not, because absolutely everyone knows this phrase. Everyone also knows the 404 number. So in any case, the user understands what happened. Although @deptyped and @LWJerri may have a different opinion?

Reply:
50/50 in my opinion, but we also must remember - docs can read people who just started learning programming and English, so...